rtengine/improcfun.cc

```diff
@@ -1857,14 +1874,16 @@ void ImProcFunctions::ciecam_02float (CieImage* ncie, float adap, int begh, int
                     }
 
                 }
+                float meanQ;
```

Comment:
@Desmis This looks like a loose end. I get this compiler warning:

```
/home/user/src/rawtherapee/rtengine/improcfun.cc: In member function ‘void rtengine::ImProcFunctions::ciecam_02float(rtengine::CieImage*, float, int, int, int, int, rtengine::LabImage*, const rtengine::procparams::ProcParams*, const rtengine::ColorAppearance&, const rtengine::ColorAppearance&, const rtengine::ColorAppearance&, LUTu&, LUTu&, LUTf&, LUTf&, float&, int, int, bool, float&, float&, float&, int)’:
/home/user/src/rawtherapee/rtengine/improcfun.cc:1894:23: warning: variable ‘meanQ’ set but not used [-Wunused-but-set-variable]
                 float meanQ;
                       ^~~~~
```

Could you take a look?

Best,
Flössie